assets/src/edit-story/elements/text/test/output.js

```diff
@@ -221,7 +249,7 @@ describe('TextOutput', () => {
         box: { width: 1080, height: 1920, x: 50, y: 100, rotationAngle: 0 },
       };
 
-      await expect(<TextOutput {...props} />).toBeValidAMPStoryElement();
+      expect(<TextOutput {...props} />).toBeValidAMPStoryElement();
```

Comment:
Unrelated change? `toBeValidAMPStoryElement` is still async

Reply:
oops! yeah, before I rebased I was missing an eslint package and deleted these in a panic. ;)